### PR TITLE
Corrections to the french translation

### PR DESCRIPTION
**Pull request by AlexisPrel**
# Description

This is an attempt at a full proofread of the French translation.
It turned out to be much more work than expected, so I will try to finish this week-end and update that PR before it's reviewed.

Even  "as is", I think this is definitely a step in the right direction.

## Issues Resolved

While it does not address specific tickets, this PR provides a number of important fixes.

Basic stuff that any native speaker would agree about:
- french-style case
- french-style punctuation for "!", "?" and ":"
- fixed syntax for gender, singular/plurals/verbal modes  (imperatives -> infinitives where appropriate)
- A -> À where appropriate

Harmonization:
- backup -> sauvegarde
- bookmark(s) -> marque-page(s) ; removed "favoris". "favoris" might be preferable,  but in any case the two words cannot coexist.
 - calendar -> calendrier, removed "agenda"

A small number of missing strings has been added.

## Check List

Open to debate:
- [x] "poll imported tasks for changes" -> "détecter les modifications des tâches importées" ; in any case "sonder" is better than "interroger" or "surveiller" in this context. The syntax of the previous translation tries too hard to follow English's logic.
- [x] "CALDAV">"FORM_SECTION" > "HELP" the first sentence is too long and contains too many pronouns, even in the English version. I re-arranged so that pronouns lie closer to the nominal group being referred to.
- [x] git "issues" are left untranslated almost everywhere, except in some places where either "problèmes" or "tickets" are used. Not sure about the best choice so I left unmodified.

### CODE REVIEW

**Comment by AlexisPrel** [comment]
Woops, meant to create that as a draft PR, note sure how to change that now.
Please wait until Monday to validate.

**Comment by AlexisPrel** [comment]
I finished harmonizing word choices, here is a quick reference:

english version|french translation | rejected alternatives
----------------|-------------------|----------------------
issue | ticket | issue, problème
calendrier | calendar | agenda
backlog | arriéré | backlog
request | requête | demande
plan | planifier, prévoir | 
 schedule | programmer | planifier, prévoir
 bookmark | marque-page | favori
 repeat task | tâches récurrentes | tâches répétées
 
 This is open to debate but should help keeping everything consistent.

**Comment by AlexisPrel** [comment]
List of known issues I have no time to fix:

- [ ] I stopped systematically checking strings from row 600 onwards, proofread is not guaranteed beyond that point and several translations are outdated
- [ ] Many times it is unclear if English sentence is infinitive or imperative. I checked in the interface for numerous instances but some may have escaped my attention. Search for pattern "ez " and "er " to address this.
- [ ] capitalization is very inconsistent, especially in patterns of the form "Xxxx : Yyyy". About 50% does "Xxxx : yyyy", the other 50% does "Xxxx: Yyyy". There are many other occasions. 
- [ ] "datetime" has no satisfying translation in French

**Comment by johannesjo** [comment]
Wow that's great @AlexisPrel ! Thank you very much! 

About your question: I think you're pretty much spot on with the list. One thing that comes to my mind is, that I was referring to the concept of "repeatable tasks" sometimes as "repeating tasks" and other times as "repeat tasks". Another concept word that comes to my mind is "reminder" which again is used inconsistent in place of schedule sometimes. Other important words might be:
* Note(s)
* Quick History
* Sync
* Global Settings
* Credentials
* Worklog
* Estimate (for the tasks)
* Timeline
* Break Reminder
* Time  Tracking Reminder
* Idle (time) handling
* Simple Counter(s)


You seem to be impressively good in spotting these things. If you ever find the time and feel like it, I would greatly appreciate your help in making things more consistent for the English base texts as well, which would likely offer benefits for every language.

**Comment by johannesjo** [comment]
Shall I merge the current state, or do you want to modify it further first?
